content/english/useless-box/learning-concepts.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Learning concepts"
+description: ""
+draft: true
+weight: 2
+---
+## How do we make a "useless" box? Let's learn some basic concepts first!
```

Comment:
```suggestion
How do we make a "useless" box? Let's learn some basic concepts first!
```

Reply:
are you suggesting to remove the heading here? I'm fine if that's preferred, but wanted to double-check